unittest_requirements.txt

```diff
@@ -2,3 +2,4 @@ mock
 pytest-django
 pulp-smash @ git+https://github.com/pulp/pulp-smash.git
 pytest<8
+pyjwkest>=1.4,<=1.4.2
```

Comment:
Do we still need this dependency?

Reply:
Hum... I probably had issues before removing the test_convert.py file:
https://github.com/pulp/pulp_container/pull/1529/files#diff-2f5092e427a2a7997e2ce75c8c252cbd9d3d30b33f2ab32dbe1b241194de98bdL6
I just tried again without it and had no dependency errors. Removing it.